docs/pages/docs/tile/select-menu-tile.mdx

```diff
@@ -115,11 +153,66 @@ FSelectMenuTile<Value>(
       const _SelectMenuTileForm({super.key});
 
       @override
-      State<_SelectMenuTile> createState() => _SelectMenuTileFormState();
+      State<_SelectMenuTile> createState() => _SelectMenuTileState();
     }
 
-    class _SelectMenuTileFormState extends State<_SelectMenuTileForm> {
-      final FRadioSelectGroupController<Notification> controller = FRadioSelectGroupController(Notification.all);
+    class _SelectMenuTileState extends State<_SelectMenuTile> {
+      final controller = FRadioSelectGroupController(Notification.all);
+
+      @override
+      Widget build(BuildContext context) => FSelectMenuTile(
+        groupController: controller,
+        autoHide: true,
+        validator: (value) => value == null ? 'Select an item' : null,
+        prefixIcon: FIcon(FAssets.icons.bell),
+        title: const Text('Notifications'),
+        details: ListenableBuilder(
+          listenable: controller,
+          builder: (context, _) => Text(
+            switch (controller.values.firstOrNull) {
+              Notification.all => 'All',
+              Notification.direct => 'Direct Messages',
+              null || Notification.nothing => 'None',
+            },
+          ),
+        ),
+        menu: [
+          FSelectTile(title: const Text('All'), value: Notification.all),
+          FSelectTile(title: const Text('Direct Messages'), value: Notification.direct),
+          FSelectTile(title: const Text('None'), value: Notification.nothing),
+        ],
+      );
+
+      @override
+      void dispose() {
+        controller.dispose();
+        super.dispose();
+      }
+    }
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+
+### Scrollable
```

Comment:
I think the scrollable example should be a lot longer. It's fairly annoying to hit the bounds almost immediately and scroll the webpage when testing the widget.